projects/eslint-config/plugins/portal/tests/lib/rules/no-metal-plugins.js

```diff
@@ -26,6 +35,15 @@ ruleTester.run('no-metal-plugins', rule, {
 				},
 			],
 		},
+		{
+			code: "import {Anim} from 'metal-anim';",
```

Comment:
Case in point: `metal-anim` [doesn't even have a named export](https://github.com/metal/metal-plugins/blob/97ae8683fabd45efadbf12a966fd9d57fcb9073f/packages/metal-anim/src/Anim.js#L97).